web/src/composebox_typeahead.js

```diff
@@ -235,6 +235,7 @@ function handle_keydown(e) {
                     ) {
                         $("#compose-send-button").prop("disabled", true);
                         compose.finish();
+                        $("#compose-send-button").prop("disabled", false);
```

Comment:
This might be an improvement, but looking at this code, I can't help but think that we should not mutate this disabled state, which looks to be a hack to prevent double-sends (which should be being enforced by the compose spinner mechanism, which already protects us from double-clicking the `send` button).